.eslintrc.js

```diff
@@ -3,14 +3,13 @@
 module.exports = {
   root: true,
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 2022,
```

Comment:
Can you drop these changes from the PR? I prefer to not include internal changes in enhancement PRs.